app/models/manageiq/providers/storage_manager/cinder_manager.rb

```diff
@@ -9,7 +9,6 @@ class ManageIQ::Providers::StorageManager::CinderManager < ManageIQ::Providers::
            :authentication_status_ok?,
            :authentications,
            :authentication_for_summary,
-           :zone,
```

Comment:
most of these are redelegated in the child
I would like to get rid of most of these delegations at a future date - so I'm not sure the best place for the delegations to live.

having overlapping list in the child and parent isn't doing anyone any favors, especially since it is in 2 different repos so making changes is that much more difficult.